resolveImports.mjs

```diff
@@ -11,8 +23,14 @@ async function exists(module) {
   }
 }
 
+/**
+ * Recursively parses and resolves a module's imports.
+ * @param {string} module The path to the module.
+ * @param {boolean} [root=true] Whether the module is the root module.
+ * @returns {Promise<ResolvedImports>} The resolved modules.
+ */
 export default async function resolveImports(module, root = true) {
-  let source;
+  let source = "";
```

Comment:
I vaguely recall that issue

Reply:
Why can't we rely on TS inferring a string here?

Reply:
Because you are creating an empty string, which has a runtime perf cost, for no reason other than to get TypeScript to work how you want. There are other ways that don't have a runtime cost.